imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```

Comment:
WHERE ARE THE TESTS!?  THE HUMANITY!  ROGUE UNTESTED DB MODELS WANDERING ABOUT IN THE WILD!!  GOD HELP US ALL!